lib/endpoints/class-wp-json-users-controller.php

```diff
@@ -208,6 +208,9 @@ public function create_item( $request ) {
 			}
 			$user->ID = $user_id;
 		}
+
+		$this->update_additional_fields_for_object( $user, $request );
```

Comment:
I don't see why we'd listen for a WP_Error, as the update to the "main" object has been done - we wouldn't want to return an error anyway presumably?